playbook/app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -92,18 +108,19 @@ const Currency = (props: CurrencyProps) => {
             dark={dark}
             size={sizes[size]}
         >
-          {`${whole}`}
+          {abbreviate ? getAbbreviatedValue('amount') : whole}
```

Comment:
We should steer towards leaving logic out of the templating area. Could you please move this to a local variable?